README.md

```diff
@@ -0,0 +1,143 @@
+# `dss-conduits`
```

Comment:
```suggestion
# `dss-allocator`
```

Reply:
oh man, I swear I fixed this like 3 times.